### PR TITLE
Track uplift ad impressions

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: app/jobs/create_impression_job.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
   queue_as :impression
 
-  def perform(id, campaign_id, property_id, ad_template, ad_theme, ip_address, user_agent, displayed_at_string)
+  def perform(*args)
+    # Protection for impressions during deploy. Can remove after successful deploy
+    args = args.insert(5, "false") if args.length == 8
+    create_impression(*args)
+  end
+
+  def create_impression(id, campaign_id, property_id, ad_template, ad_theme, uplift, ip_address, user_agent, displayed_at_string)
```
Comment (hopsoft) [inline app/jobs/create_impression_job.rb]: Might be safer to tack `uplift` to the end of the args and default it to `false`. Thoughts?

File: app/jobs/create_impression_job.rb [hunk 2]
```diff
@@ -23,6 +29,7 @@ def perform(id, campaign_id, property_id, ad_template, ad_theme, ip_address, use
       property: property,
       ad_template: ad_template,
       ad_theme: ad_theme,
+      uplift: uplift == "true",
```
Comment (hopsoft) [inline app/jobs/create_impression_job.rb]: Should we call `uplift.to_s` here?

File: app/jobs/create_impression_job.rb (hunk 1 above)
Comment (coderberry) [reply app/jobs/create_impression_job.rb]: oh.. good call!

File: app/jobs/create_impression_job.rb (hunk 2 above)
Comment (coderberry) [reply app/jobs/create_impression_job.rb]: yes

**Pull request merged**